mmdet/models/backbones/csp_darknet.py

```diff
@@ -115,7 +115,11 @@ def __init__(self,
 
     def forward(self, x):
         x = self.conv1(x)
-        x = torch.cat([x] + [pooling(x) for pooling in self.poolings], dim=1)
+        if x.dtype == torch.float16:
+            x = x.float()
+            x = torch.cat([x] + [pooling(x) for pooling in self.poolings], dim=1).half()
+        else:
+            x = torch.cat([x] + [pooling(x) for pooling in self.poolings], dim=1)
```

Comment:
```suggestion
        with torch.autocast(enabled=False):
            x = torch.cat([x] + [pooling(x) for pooling in self.poolings], dim=1)
```

Reply:
can we simplified the modification as the suggestion?

Reply:
looks good to me